index.js

```diff
@@ -25,41 +23,13 @@ export default [
     },
 
     rules: {
+      // core rules
       'array-callback-return': 'error',
       'block-scoped-var': 'error',
       curly: 'error',
       'default-case': 'error',
-
-      'dot-notation': [
-        'error',
-        {
-          allowPattern: '^[a-z]+(_[a-z]+)+$',
-        },
-      ],
```

Comment:
I left these and the rules below in place. I just organized them into core and plugin groups.